src/main/resources/application-local.properties

```diff
@@ -0,0 +1,12 @@
+spring.liquibase.change-log=db/changelog/liquibase-changelog.xml
+spring.datasource.url=jdbc:mysql://localhost:3306/auditdb
+spring.datasource.username=root
+spring.datasource.password=PaulIsrael1609
```

Comment:
you mustn't store any sensitive data in the repository even for the local config